Changelog.md

```diff
@@ -106,12 +106,73 @@ This changelog covers the entire dave project which includes a few repositories,
+
+- Mud Plugin
+	- Demonstration and associated models for incorporation of the Gazebo mud plugin into the dave environment [dave PR #184](https://github.com/Field-Robotics-Lab/dave/pull/184)
+	- Documentation and tutorial - updates Occlusion documentation to include the mud demo [wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Occlusion)
+
+- Bimanual Manipulation
+  - Creation of MoveIt-based infrastructure for motion planning with single and multi-arm systems. [uuv_manipulators PR #8](https://github.com/Field-Robotics-Lab/uuv_manipulators/pull/8)
+  - Addition and configuration of multiple arms onboard arbitrary robot. [dave PR #206](https://github.com/Field-Robotics-Lab/dave/pull/206)
+  - Demonstration of bimanual manipulation scenarios using joy teleop, RVIZ interaction, and MoveIt ROS nodes.  [dave PR #228](https://github.com/Field-Robotics-Lab/dave/pull/228)
+  - Documentation and tutorial [wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Bimanual-Manipulation-Setup-and-Examples) 
+
+- Mating Plugin
+  - Expose mating forces for mating plugin as ROS publication [dave PR #189](https://github.com/Field-Robotics-Lab/dave/pull/189)
+  - Electrical flying lead demonstration scenario - incremental improvement [dave PR #161](https://github.com/Field-Robotics-Lab/dave/pull/161)
+  - Updated [Electrical Plug Mating Plugin Wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Electrical-Plug-Mating-Plugin)
+
+- [Building Bathymetry Heightmaps Wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Building-a-simple-underwater-environment-via-heightmaps)
+
+- Object model degradation tool [wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Object-Degradation-Tools) documentation
+
+- Object Models
+  - New 3D object models. dave PRs [#194](https://github.com/Field-Robotics-Lab/dave/pull/194), [#205](https://github.com/Field-Robotics-Lab/dave/pull/205), [#209](https://github.com/Field-Robotics-Lab/dave/pull/209), and [#225](https://github.com/Field-Robotics-Lab/dave/pull/225)
 
 ### Coming Soon
 
 - Manipulator reaction force interface
-- Electrical flying lead demonstration scenario - incremental improvement [PR#161](https://github.com/Field-Robotics-Lab/dave/pull/161)
+
+- Bathymetry integration plugin usability upgrades [Dave PR #180](https://github.com/Field-Robotics-Lab/dave/pull/180)
+  - Simpler latitude/longitude coordinate system
+  - A new script to set the initial position with geodetic coordinates
+  - GPS Viewer\Logger and a new flag to set how many tiles to keep in the scene
+  - New tutorial document and demo files [Bathymetry integration wiki](https://github.com/Field-Robotics-Lab/dave/wiki/Bathymetry-Integration)
+
+- Example and tutorial for creating new seafloor heightmaps from arbitrary-format bathymetry sources [wiki] (https://github.com/Field-Robotics-Lab/dave/wiki/DRAFT:-Building-a-simple-underwater-environment-via-heightmaps)
```

Comment:
New URL

Consider adding https://github.com/Field-Robotics-Lab/dave/pull/207

```suggestion
- Example and tutorial for creating new seafloor heightmaps from arbitrary-format bathymetry sources [wiki] (https://github.com/Field-Robotics-Lab/dave/wiki/Building-a-simple-underwater-environment-via-heightmaps)
```

Reply:
If the demo stops seg faulting before we close this up for the day, I will move all from the "coming soon" to the main changelog.